onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -2493,7 +2581,7 @@ CUDAExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
       continue;
 
     const auto& node = *p_node;
-    if (!node.GetExecutionProviderType().empty()) {
+    if (!node.GetExecutionProviderType().empty() && node.Domain() != kMSInternalNHWCDomain) {
```

Comment:
Can you please explain this change ? What sets the node's domain ? Is it the layout optimizer ?